src/app/password-recovery/pages/password-recovery/password-recovery.component.html

```diff
@@ -1,122 +1,132 @@
 <main>
```

Comment:
Attribute id is missing

Reply:
This is added on this PR https://github.com/ORCID/orcid-angular/pull/228/files